pravega/pravega
# hostStoreType conf docker

**Change log description**
HostStore configurable via an environment variable to pravega controller docker image.

**Purpose of the change**
https://github.com/emccode/pravega/issues/463

**What the code does**
Configures HostStore Type as an environment variable which defaults to InMemory if not set.

**How to verify it**
```
./gradlew buildPravegaControllerImage
docker run --net=host -d  -e "HOST_STORE_TYPE=Zookeeper" pravega/pravega-controller:0.0-PRERELEASE
```
See controller logs in container to see config value being set or call some controller API via rest and see appropriate entries in ZK.
